projects/hometown_homepage/styles.css

```diff
@@ -0,0 +1,154 @@
+/**
+Palette: https://scrimba.com/links/hometown-palette
+RED: #E63946
+LIGHT: #F1FAEE
+AQUA: #A8DADC
+LIGHT BLUE: #457B9D
+DARK BLUE: #1D3557
+
+*/
+
+body {
+  margin: 0;
+  text-align: center;
+}
+
+img {
+  width: 100px;
+}
+.home{
+  padding-right: 300px;
+  color: gray;
+}
+.main{
+  height: 700px;
+  width: 500px;
+  margin-left: 450px;
+}
```

Comment:
Don't set a width or height. Make it take the whole screen/page width and height.